service/src/main/java/com/commercetools/pspadapter/payone/mapping/CreditCardRequestFactory.java

```diff
@@ -36,7 +36,7 @@ public CreditCardPreauthorizationRequest createPreauthorizationRequest(
         final String pseudocardpan = ctPayment.getCustom().getFieldAsString(CustomFieldKeys.CARD_DATA_PLACEHOLDER_FIELD);
         CreditCardPreauthorizationRequest request = new CreditCardPreauthorizationRequest(getPayoneConfig(), pseudocardpan, paymentWithCartLike);
 
-        mapFormPaymentWithCartLike(request, paymentWithCartLike);
+        mapFormPaymentWithCartLike(request, paymentWithCartLike, false);
```

Comment:
I would prefer to assign this boolean value into a variable called `isIgnoreShippingAdress` before parsing it into another method, so that developers can quickly identify what true/false represents here.

Reply:
399a3fe08